railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -28,3 +28,7 @@
 # Do not treat an `ActionController::Parameters` instance
 # as equal to an equivalent `Hash` by default.
 # Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
+
+# Active Record Encryption now uses SHA-256 as its hash digest algorithm. Important: If you have data encrypted
+# with previous versions, you should not set the new default or the existing data will fail to decrypt.
+# Rails.application.config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA256
```

Comment:
I don't think this is sufficient. If you have data encrypted with previous versions, then you should set the *old* version as a config. That way if you later do `load_defaults 7.1` it won't change this under the hood.

```suggestion
# Active Record Encryption hash digest algorithm: 2 options
#
# The new default hash digest algorithm is SHA-256. If you haven't used Active Record Encryption
# yet, it is safe to upgrade.
#
# **Important**: If you have data encrypted already, it is likely encrypted using SHA-1. You should keep the old default or the existing data will fail to decrypt.
#
# New default:
# Rails.application.config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA256
#
# Old default:
# Rails.application.config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA1
```

Reply:
Good point @ghiculescu. I'll review this and rebase the branch shortly.

Reply:
I think your changes look good 👍